packages/react-components/src/organisms/UserProfileRole.tsx

```diff
@@ -131,6 +137,21 @@ const UserProfileRole: React.FC<UserProfileRoleProps> = ({
           )}
         </div>
       )}
+      {(reachOut || isOwnProfile) && role === 'Staff' && (
```

Comment:
Can we remove `reachOut` from this condition as it later checks for `reachOut` on line 145?

```suggestion
      {isOwnProfile && role === 'Staff' && (
```

Reply:
It wouldn't show `reachOut` if  `isOwnProfile` is false